HomeWork9/HomeWork9/Action.cs

```diff
@@ -0,0 +1,32 @@
+﻿using Abstracts;
+using Models.Enum;
+using Services.Abstracts;
+
+namespace HomeWork9
+{
+    internal class Action : IAction
+    {
+        private readonly ILoggerService _logger;
+
+        public Action(ILoggerService loggerService)
+        { 
+            _logger = loggerService;
+        }
+
+        public void StartMethod() => _logger.Log(LogType.Info, "Start method");
+
+        public void BusinessException()
+        {
+            _logger.Log(LogType.Warning, "Skipped logic in method");
+
+            throw new BusinessException();
+        }
+
+        public void ThrowExeption()
+        {
+            _logger.Log(LogType.Error, "I broke a logic");
+
+            throw new Exception();
+        }
```

Comment:
We can just create method that will accept exception and message instead of create two metods

Reply:
It`s criterial of task)
The methods in the Actions class must call the logger to log a message containing the text :
1 method “Start method:” and the name of this method. This log must be of type Info. 
2 method The method throws a BusinessException with the text “Skipped logic in method”
3 method The method throws an Exception with the text “I broke a logic”.